gremlin-python/src/main/python/gremlin_python/structure/io/graphsonV4.py

```diff
@@ -69,6 +69,10 @@ def typed_value(cls, type_name, value, prefix="g"):
     def format_type(cls, prefix, type_name):
         return "%s:%s" % (prefix, type_name)
 
+    @classmethod
+    def write_label(cls, value, writer):
```

Comment:
I'm not sure how useful this function is, but I think the code is a little clearer because of it?